tests/helpers/checkboxTreeHelper-test.js

```diff
@@ -0,0 +1,23 @@
+import createCheckboxTreeDataStrucure from 'helpers/checkboxTreeHelper';
+import {
+    naicsMock2,
+    naicsMockCleanData,
+    naicsMockCleanDataInitialLoad
+} from '../containers/search/filters/naics/mockNAICS';
+
+describe('CheckboxTreeHelper', () => {
+    const keysToBeMapped = {
+        value: 'naics',
+        label: 'naics_description'
+    };
+    it('should map API response to react-checkbox-tree structure', () => {
```

Comment:
Shouldn't we have a nested describe here.

describe(fileName)... describe(methodName)

Reply:
i mean, there is only one function, and two test cases. so no. maybe if we had other functions in here.